### DISCUSSION TITLE
regarding the client_exec_interactive example

### DISCUSSION THREAD

**Discussion by Souldiv** [Eugeny/russh#276] [Q&A] [answered]
Hi! I ran the example given with the following command, but the after executing the command it exits out. I was under the impression it should create an interactive ssh session? am I incorrect?

```
cargo run --all-features --example client_exec_interactive -- -k ~/.ssh/testkey -u root -p 2222 localhost ls
```

**Comment by Eugeny** [comment, marked as answer]
Use "bash" instead of "ls" for a shell, or request_shell instead of request_exec
